### PR TITLE
[PJRT] Update pjrt.md for new release

### PR DESCRIPTION
**Pull request by will-cromar**

### CODE REVIEW

**Review by JackCaoG — APPROVED** [review]

**Comment by JackCaoG** [comment]
Please also cherry-pick to 2.0 release

File: docs/pjrt.md [hunk 1]
```diff
@@ -1,26 +1,127 @@
...
-runtime to the [PjRt
+runtime to the [PJRT
 runtime](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/compiler/xla/pjrt)
 used by [JAX](https://github.com/google/jax).
 
-PjRt is available as an _experimental preview_ in PyTorch/XLA r1.13. The
-PyTorch/XLA team will provide limited support on a best-effort basis during this
-preview. If you encounter a bug with PjRt, please file an issue on GitHub with
-the `runtime` tag.
+PJRT is available for preview in PyTorch/XLA r2.0. **We are planning to make
```
Comment (JackCaoG) [inline docs/pjrt.md:12]: we actually call PJRT stable in our blog post, maybe we should align on the naming. @miladm

Comment (JackCaoG) [reply docs/pjrt.md:12]: Well, it is the only feature that we could call stable otheriwise everything we introduced are preview lol.

Comment (will-cromar) [reply docs/pjrt.md]: Discussed offline and decided to use "Beta"

**Review by alanwaketan — APPROVED** [review]
LGTM.

**Review by Liyang90 — COMMENTED** [review]
LGTM

**Pull request merged**